config/web-app/istio.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: models-web-app
+spec:
+  gateways:
+  - knative-ingress-gateway.knative-serving
```

Comment:
Can this be a variable `$(ingressGateway)` that uses the  `config/default/params.env` setting? With this, you wont need the VS gateway patch in the kubeflow overlay.  

Looking at `config/default/params.env` now, in order for you to use it, the following change ro params.env will need to be made: 

```
ingressGateway=knative-ingress-gateway.knative-serving
```

Then the `config/configmap/inferenceservice.yaml` will need the ingress section changed to:
`"ingressGateway" : "$(ingressGateway)"` 

Similarly, the quotes and comma will have to be removed from the kubeflow overlay params.env.

Probably should have been this way in the first place.

Reply:
Yes I would also prefer to use the `$(ingressGateway)` var, as it would simplify the manifests and we won't need to maintain an extra patch. Also thanks for the instructions!

I'll add a commit with the proposed refactor. I'll also give a heads up here if something comes up that we didn't catch with our first understanding.

Reply:
Pushed a commit with the described changes. Building both the `default` and the `kubeflow` artifacts results in using the correct gateway both in the VirtualSercice and in the ConfigMap.

I also removed the extra patch I had introduced to edit this field